Dockerfile

```diff
@@ -18,9 +18,6 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     libxml2-dev \
     libssl-dev \
     libssh2-1-dev \
-    libsnappy-dev \
```

Comment:
This change made me realize that we need to make a fix to how we're doing versioning of our docker files.  For new users to actually see these files removed in the container they are using,
-A team member needs to push the new container version to Docker Hub and update the DOCKER_VERSION file
-OR the user will have to build the container locally by running tools/builddocker.sh

- For testing purposes, you will need to build the new container with your changes by running tools/builddocker.sh
- Either @anmolah  or myself will validate the changes, approve the commit and then submit a separate PR for bumping the version number after updating DockerHub with your container.  This is temporary/a workaround and the actual solution will be tracked in #17